ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -3028,3 +3064,34 @@ SDBPC:
   - supply_air_damper_percentage_sensor
   implements:
   - MONITORING
+
+HWVM:
+  description: "Heating water valve monitoring on discharge side."
+  is_abstract: true
+  opt_uses:
+  - heating_thermal_power_capacity
+  - heating_water_flowrate_sensor
+  uses:
+  - heating_water_valve_percentage_sensor
+  - heating_water_valve_percentage_command
+  implements:
+  - OPERATIONAL
+
+CHWVM:
+  description: "Chilled water valve monitoring on discharge side."
+  is_abstract: true
+  opt_uses:
+  - cooling_thermal_power_capacity
+  - chilled_water_flowrate_sensor
+  uses:
+  - chilled_water_valve_percentage_sensor
+  - chilled_water_valve_percentage_command
+  implements:
+  - OPERATIONAL
+
+DMM:
+  description: "Device mode monitoring."
+  is_abstract: true
+  uses:
+  - device_mode
```

Comment:
This feels like a catch-all.  Unless this is a HVAC specific field (and we're trying to avoid non-global fields until there's full backend support for them), won't we end up with a massive fan-out of states for this unless we make it a little more specific?

Reply:
adjusted to run_mode